sdk/resourcemanager/Azure.ResourceManager.Core/samples/Sample1_HelloWorld.md

```diff
@@ -2,26 +2,32 @@
 
 >Note: Before getting started with the samples, make sure to go trough the [prerequisites](https://github.com/Azure/azure-sdk-for-net/tree/feature/mgmt-track2/sdk/resourcemanager/Azure.ResourceManager.Core#prerequisites).
 
+Namespaces for this example:
+```C# Snippet:Hello_World_Namespaces
+using System;
```

Comment:
don't we need Azure.ResourceManager.Core?

Reply:
Yes, good catch.